system/iptables.py

```diff
@@ -278,6 +282,8 @@ def construct_rule(params):
         append_param(rule, ','.join(params['ctstate']), '--state', False)
     append_limit(rule, params['limit'])
     append_param(rule, params['limit'], '--limit', False)
+    if params['limit']:
```

Comment:
Just because right now the `-m limit` param is appended to the rule only if the `limit` param is set.

Another option would be to change the `append_limit` method to set the `-m limit` param if `limit` is set or if `limit_burst` is set.